travis-ci/vmtest/run_selftests.sh

```diff
@@ -28,6 +28,12 @@ test_verifier() {
 	travis_fold end test_verifier
 }
 
+test_bpftool() {
+	travis_fold start test_bpftool "Testing bpftool"
```

Comment:
bpftools_checks "bpftool checks" ? "check" doesn't mean "run and test", so seems fitting.

Also note that we have libbpf CI  at https://github.com/libbpf/libbpf which uses very similar set of tests, so would be good to have it there as well, I suppose.

Reply:
Sounds good, I changed to “bpftool checks”.

Yes, I remember you pointed to libbpf's CI too. I was planning to have the current PR merged here first, and to follow up with libbpf/libbpf, but given that the files are so similar I could understand if you prefer to merge both at the same time. So I opened a similar PR at https://github.com/libbpf/libbpf/pull/352.